### PR TITLE
Rewinding NettyResponse's BodyPart ByteBuffer

### PR DESCRIPTION
**Pull request by marenzo**
Related to #1212

### CODE REVIEW

File: client/src/main/java/org/asynchttpclient/netty/NettyResponse.java [hunk 1]
```diff
@@ -179,6 +179,7 @@ public ByteBuffer getResponseBodyAsByteBuffer() {
         for (HttpResponseBodyPart part : bodyParts)
             target.put(part.getBodyPartBytes());
 
+        target.rewind();
```
Comment (slandelle) [inline client/src/main/java/org/asynchttpclient/netty/NettyResponse.java]: Why not flip?

Comment (marenzo) [inline client/src/main/java/org/asynchttpclient/netty/NettyResponse.java]: does it matter? I don't mind to change to flip, but it's doesn't do anything special in this context - it's updates the state of limit to be the same as already defined (which is the buffer length), while rewind just sets back to the index to the beginning.

Comment (slandelle) [inline client/src/main/java/org/asynchttpclient/netty/NettyResponse.java]: Well, `flip` is the idiomatic way for switching from write to read and vice versa, which is exactly what we need here.

Comment (marenzo) [inline client/src/main/java/org/asynchttpclient/netty/NettyResponse.java]: done.

**Comment by slandelle** [comment]
Thanks!

**Pull request merged**